qa/tasks/mgr/dashboard/test_cephfs.py

```diff
@@ -199,3 +220,28 @@ def test_snapshots(self):
         self.rm_dir('/movies/dune/extended_version')
         self.rm_dir('/movies/dune')
         self.rm_dir('/movies')
+
+    def test_quotas_default(self):
```

Comment:
This method seems to be unused.

Reply:
This is method is executed by Teuthology.